src/compiler/crystal/semantic/call_error.cr

```diff
@@ -396,7 +396,7 @@ class Crystal::Call
   end
 
   def def_full_name(owner, a_def, arg_types = nil)
-    Call.def_full_name(owner, a_def, arg_types = nil)
+    Call.def_full_name(owner, a_def, arg_types)
```

Comment:
The code transformation would be to ignore arg_types and leave `nil` but it seems weird. It should be narrowed a bit down what is the expected thing to do here / add missing spec.